tests/Cache/CacheRepositoryTest.php

```diff
@@ -149,10 +148,12 @@ public function testCacheAddCallsRedisStoreAdd()
 
     public function dataProviderTestGetMinutes()
     {
+        Carbon::setTestNow(Carbon::parse($this->getTestDate()));
+
         return [
             [Carbon::now()->addMinutes(5)],
-            [(new DateTime('2030-07-25 12:13:14 UTC'))->modify('+5 minutes')],
-            [(new DateTimeImmutable('2030-07-25 12:13:14 UTC'))->modify('+5 minutes')],
```

Comment:
it is okay to declare hardcoded strings in tests.

Reply:
It would be, but now the same string has to be repeated in 2 different methods and in my opinion this is too much

Reply:
that isn't too much. That is quite normal for tests. You are rarely going to change all the dates at once.